types/evidence_test.go

```diff
@@ -33,6 +33,45 @@ func TestEvidenceList(t *testing.T) {
 	assert.False(t, evl.Has(&DuplicateVoteEvidence{}))
 }
 
+// TestEvidenceListProtoBuf to ensure parity in protobuf output and input
+func TestEvidenceListProtoBuf(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	const chainID = "mychain"
+	ev, err := NewMockDuplicateVoteEvidence(ctx, math.MaxInt64, time.Now(), chainID)
+	require.NoError(t, err)
+	data := EvidenceList{ev}
+	_ = data.ByteSize()
```

Comment:
does this method have side effects? (that seems suspicious)

Reply:
This was a copy paste from what was there for EvidenceData. My guess is that there it made sense to run it as this function would actually populate the previously existing byteSize field of the EvidenceData. Does not make sense to call it here as this only returns a value w/o populating anything.